hubgrep/frontend_blueprint/templates/components/search_form/_search_form.scss

```diff
@@ -50,6 +50,15 @@
     }
   }
 
+  .filter-item {
+    display: flex;
+    align-items: center;
+
+    input {
+      margin-right: 0.5em;
+    }
+  }
+
```

Comment:
centers filter item parts, and fixes odd linebreaks within an item on hoster filters (now moves the next item to new row)